### PR TITLE
Change TumorColumnFormatter to support structural variant alterations (two genes)

### PR DESCRIPTION
**Pull request by alisman**
example patient: https://genie-private.cbioportal.org/patient/summary?studyId=nsclc_genie_bpc&caseId=GENIE-DFCI-004022

Before: 
image

After:
image

Fixes https://github.com/cbioportal/cbioportal/issues/9300

### CODE REVIEW

File: src/pages/patientView/structuralVariant/PatientViewStructuralVariantTable.tsx [hunk 1]
```diff
@@ -85,18 +85,23 @@ export default class PatientViewStructuralVariantTable extends React.Component<
...
                 columns.push({
                     name: 'Samples',
-                    render: (d: StructuralVariant[]) =>
-                        TumorColumnFormatter.renderFunction(
+                    render: (d: StructuralVariant[]) => {
+                        return TumorColumnFormatter.renderFunction(
                             d.map(datum => ({
                                 sampleId: datum.sampleId,
-                                entrezGeneId: datum.site1EntrezGeneId,
+                                entrezGeneId: [
```
Comment (alisman) [inline src/pages/patientView/structuralVariant/PatientViewStructuralVariantTable.tsx:92]: @inodb safe to assume they all have site1 and site2?

Comment (inodb) [reply src/pages/patientView/structuralVariant/PatientViewStructuralVariantTable.tsx:92]: Some might have only one gene (a lot of them can be outside of genes). For data import at least either site 1 or site 2 needs to be specified

File: src/pages/patientView/structuralVariant/PatientViewStructuralVariantTable.tsx [hunk 2]
```diff
@@ -85,18 +85,33 @@ export default class PatientViewStructuralVariantTable extends React.Component<
...
-                            d.map(datum => ({
-                                sampleId: datum.sampleId,
-                                entrezGeneId: datum.site1EntrezGeneId,
-                            })),
+                    render: (d: StructuralVariant[]) => {
+                        return TumorColumnFormatter.renderFunction(
+                            d.map(datum => {
+                                // if both are available, return both genes in an array
+                                // otherwise, return whichever is available
+                                const genes =
```
Comment (alisman) [inline src/pages/patientView/structuralVariant/PatientViewStructuralVariantTable.tsx:93]: @inodb issue you raised is fixed here

**Review by inodb — APPROVED** [review]
Sweet!

**Pull request merged**